tests/multi_asic_dropstat_test.py

```diff
@@ -0,0 +1,59 @@
+import os
+import sys
+import shutil
+from utils import get_result_and_return_code
+
+test_path = os.path.dirname(os.path.abspath(__file__))
+modules_path = os.path.dirname(test_path)
+scripts_path = os.path.join(modules_path, "scripts")
+sys.path.insert(0, test_path)
+sys.path.insert(0, modules_path)
+
+dropstat_path = "/tmp/dropstat-27"
+
+dropstat_masic_result="""\
+       IFACE    STATE    RX_ERR    RX_DROPS    TX_ERR    TX_DROPS    DEBUG_0    DEBUG_2
+------------  -------  --------  ----------  --------  ----------  ---------  ---------
+   Ethernet0        U         0           0         0           0          0          0
+   Ethernet4        U         0           0         0           0          0          0
+Ethernet-BP0        U         0        1000         0           0        800        100
+Ethernet-BP4        U         0        1000         0           0        800        100
+
+          DEVICE    DEBUG_1
+----------------  ---------
+sonic_drops_test       1000
+"""
+
+class TestMultiAsicDropstat(object):
+   @classmethod
+   def setup_class(cls):
+      if os.path.exists(dropstat_path):
+         shutil.rmtree(dropstat_path)
+      os.environ["PATH"] += os.pathsep + scripts_path
+      os.environ["UTILITIES_UNIT_TESTING"] = "1"
+      os.environ["UTILITIES_UNIT_TESTING_TOPOLOGY"] = "multi_asic"
+      print("SETUP")
+
```

Comment:
can you add edge cases like:
1. when no namespace is given for multi-asic
2. when namespace is given on single-asic

Reply:
See latest change. Above two edges have been covered in new UTs.